.github/workflows/ci.yml

```diff
@@ -5,7 +5,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.9', 'pypy3.10', '3.11', '3.12']
+        # Python 3.7 is the highest available for actions/setup-python
```

Comment:
Hi there @jayvdb, what does this mean?

Reply:
https://github.com/actions/setup-python supports py3.7+.  It doesnt support py3.6

Reply:
I'll take your word for taht and change "highest" to "lowest".